### ISSUE TITLE
tests: develop and write tests for steps/incremental_clustering.py

### ISSUE THREAD

**Issue by anujsinha3** [uw-ssec/MAWpy#36]

**Comment by qzchen-uw** [comment]
If a user starts with a csv file, they can also use this code, right? in other words, they do not need to first pre-process it into pandas data frame before processing it? Also, is the code limited by the data size?

**Comment by anujsinha3** [comment]
Yes, they can use the csv file. The code will take care of conversion the csv file into pandas dataframe.

More than the code itself, it depends on the machine that is running the code about its data handling capacity.
